fly/eventstream/render.go

```diff
@@ -37,6 +37,10 @@ func Render(dst io.Writer, src eventstream.EventStream, options RenderOptions) i
 			dstImpl.SetTimestamp(e.Time)
 			fmt.Fprintf(dstImpl, "%s", e.Payload)
 
+		case event.SelectedWorker:
+			dstImpl.SetTimestamp(e.Time)
+			fmt.Fprintf(dstImpl, "\x1b[1mselected worker: %s\n", e.WorkerName)
```

Comment:
nit: It will be hard to tell which step selected which worker within an `in_parallel` step. Once the origin ID has been added, we could use it to retrieve the step name from the build plan, but there isn't great support in `go-concourse` for doing that kind of thing right now, so I'm OK with skipping it for this PR.

This is a kind of general problem with the `fly watch`/`fly execute` output - it might be cool to have a [`buildkit`-style UI](https://github.com/moby/buildkit) someday so all events can be associated to their origin in the output. 🤔

Reply:
Okey, I'm going to skipping that for this PR.